kernel/comps/pci/src/arch/x86/mod.rs

```diff
@@ -30,6 +49,11 @@ fn encode_as_port(location: &PciDeviceLocation) -> u32 {
         | (((location.function as u32) & 0b111) << 8)
 }
 
+pub(crate) fn init() {
```

Comment:
Move `init` below `has_pci_bus` to be consistent with the non-x86 architectures.